src/components/Vote/Vote.jsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
-import VoteApp from 'webpack.vote';
 import Container from '../Container/Container';
 import 'webpack.vote/dist/style.min.css';
 
-const Vote = ({ section, page }) => (
-  <Container className="vote markdown">
-    <VoteApp
-      development={ false } />
-  </Container>
-);
-
-export default Vote;
+export default class Vote extends React.Component {
+  state = {
+    VoteApp: null
+  }
+  componentDidMount() {
+    if (window && !window.__ssgrun) {
+      import('webpack.vote').then(VoteApp => {
```

Comment:
it wasnt rendered server side anyway so we only import this client side dynamically